viewer/packages/pointclouds/src/styling/AnnotationListPointCloudObjectCollection.ts

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import { StylableObjectCollection } from './PointCloudObjectCollection';
```

Comment:
Mismatching classname and filename. I suggest we stick with `PointCloudObjectCollection`